File: API Buena/SistemaEsmeralda/SistemaEsmeralda.Common/Models/ClienteViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaEsmeralda.Common.Models
{
    public class ClienteViewModel
    {
        [NotMapped]
        public string Clie_DNI { get; set; }
        public int Clie_Id { get; set; }
        public string Clie_Nombre { get; set; }
        public string Clie_Apellido { get; set; }
        public DateTime Clie_FechaNac { get; set; }
        public string Clie_Sexo { get; set; }
        public string Muni_Codigo { get; set; }
        public int Esta_Id { get; set; }
        public int? Clie_UsuarioCreacion { get; set; }
        public DateTime? Clie_FechaCreacion { get; set; }
        public int? Clie_UsuarioModificacion { get; set; }
        public DateTime? Clie_FechaModificacion { get; set; }
        public bool? Clie_Estado { get; set; }
        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }


        [NotMapped]
        public string Muni_Municipio { get; set; }
        [NotMapped]
        public string Esta_EstadoCivil { get; set; }
    }
}

File: sakai-ng-master/src/app/Models/ClienteViewModel.ts
export class Cliente{
    clie_Id?:String;
    clie_Nombre?:String;
    clie_Apellido?:String;
    clie_FechaNac?:String;
    clie_Sexo?:String;
    estado_Civil?:String;
    carg_Cargo?:String;
    municipio?:String;
    Muni_Municipio:string;
}

export class ClienteEnviar {
    Clie_Id: string;
    Clie_Nombre: string;
    Clie_Apellido: string;
    Clie_FechaNac: string;
    Clie_Sexo: string;
    Muni_Codigo: string;
    Esta_Id: string;

}


export class Fill {
    clie_Id: string;
    clie_Nombre: string;
    clie_Apellido?:String;
    clie_FechaNac?:String;
    clie_Sexo?:String;
    muni_Codigo?:String;
    muni_Municipio?:String;
    carg_Cargo: string;
    esta_Id?:String;
    esta_EstadoCivil?:String;

    depa_Codigo?:String;
    depa_Departamento?:String;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}